_posts/2019-08-02-Rates-Rebates-Recommendations-Report.md

```diff
@@ -0,0 +1,276 @@
+
+User acceptance testing (UAT) for the Rates Rebate Beta began on Tuesday 21 May. The Beta trial with ratepayers started on Wednesday 5 June and ended on Monday 1 July. Councils continued to process online applications up to 1 August 2019.
+
+Funding for the Beta was approved by the Service Innovation Working Group (SIWG) within the Digital Government Partnership supported by the Government Chief Digital Officer (GCDO). To deliver the Beta, a virtual team was created with support and participation from the Service Delivery and Operations (SDO) team (who continue to provide advice and subject matter expertise), and the Government Information Services (GIS) team (who made a significant contribution to the initiative by contributing two developers at high capacity for an extended period). At this stage there has been no funding approved to proceed beyond Beta.
+
+### Current factors impacting customers using the service
+
+- **Low income, and time poor**
+The current application process is complex, takes a long time, and places an unjustified burden on many people by asking for more information than is necessary.
+- **Issues affecting mobility and access**
+The lack of an alternative, accessible way to make an application means many people who have a disability e.g. a vision impairment ([over 11% of those over 65](http://archive.stats.govt.nz/browse_for_stats/health/disabilities/DisabilitySurvey_HOTP2013/Commentary.aspx)) or a physical disability ([49% of those over 65](http://archive.stats.govt.nz/browse_for_stats/health/disabilities/DisabilitySurvey_HOTP2013/Commentary.aspx#impairtype) struggle to apply for help they are eligible to receive.
+
+Due to many (in the 2017/18 rating year 79% of all applicants were receiving New Zealand Superannuation) applicants being over 65 and/or living with a disability, the requirement to sign a Statutory Declaration in front of a witness impacts accessibility for the core customer group.
+
+**Note**: During the Beta trial, a separate paper Why remove the statutory declaration was written to support the case for its removal. This paper details the substantial benefits associated with removing the statutory declaration, when this is supported with adoption of an online application process.
+
+## Applications submitted during the trial
+
+The beta was run during the quiet June period directly following the normal “reminder” sent out by councils before the end of the financial year. Council’s indicated that this timing would work well because staff would be available to collaborate, co-design, and test improvements to the application and process. This worked well, with the design team able to work closely with both frontline and back office staff to observe processes and current practice. Council staff were also available during a two week period of user acceptance testing, which was important (we had received feedback from Tauranga that insufficient time had been allowed for user acceptance testing in the Alpha).
```

Comment:
```suggestion
We choose to run the beta during the quiet June period directly following the normal “reminder” sent out by councils before the end of the financial year. Council’s indicated that this timing would work well because staff would be available to collaborate, co-design, and test improvements to the application and process. This worked well, with the design team able to work closely with both frontline and back office staff to observe processes and current practice. Council staff were also available during a two week period of user acceptance testing, which was important (we had received feedback from Tauranga that insufficient time had been allowed for user acceptance testing in the Alpha).
```